spec/serializers/todo_serializer_spec.rb

```diff
@@ -26,6 +28,6 @@
   end
 
   it 'has the correct url' do
-    expect(subject.attributes[:url]).to eq "#{ENV.fetch('HOST_URL')}/#{todo.id}"
+    expect(subject.attributes[:url]).to eq "#{ENV.fetch('HOST_URL')}#{todo_path(todo)}"
```

Comment:
We shouldn't replicate the implementation/functionality in the specs. Please assess the explicit URL.

Reply:
My thought was that the HOST_URL can be configurable in any environment (including TEST). By changing the env variable would cause the test to fail. So I think we should still keep the HOST_URL environment variable fetch. The rest could be hard coded.

Reply:
If you think the host might change, I suggest testing only the URL path.